packages/lib/WebDavApi.js

```diff
@@ -352,6 +352,10 @@ class WebDavApi {
 			if (method === 'PUT') headers['Content-Type'] = 'text/plain';
 		}
 
+		if (!headers['Cache-Control']) {
+			headers['Cache-Control'] = 'no-store';
```

Comment:
Actually since it's a workaround for a server bug, could you please add a comment and link to the original issue?